### PR TITLE
Setup a page to host the surveygizmo survey

### PR DESCRIPTION
**Pull request by adamlofting**
Part of #1106 

This adds the survey page so we can look at it on staging, but it doesn't include this page in the donor flow yet.

R? @alicoding or @ScottDowne

### CODE REVIEW

**Comment by ScottDowne** [comment]
Looks good. I see an extra whitespace in <header /> and I wonder why make
surveygizmo a separate component.

File: pages/survey.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import React  from 'react';
+import Footer from '../components/footer.jsx';
+import Header from '../components/header.jsx';
+import SurveyGizmo from '../components/surveygizmo.jsx';
+import { IntlMixin } from 'react-intl';
+
+var ThankYou = React.createClass({
```
Comment (alicoding) [inline pages/survey.jsx]: Just module.exports here?

File: pages/survey.jsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  render: function() {
+    var className = "row new-flow-test";
+    var language = this.props.locales[0];
+    if (this.props.test) {
+      className += " " + this.props.test;
+    }
+    return (
+      <div className={className}>
+        <Header />
+        <SurveyGizmo language={language}/>
```
Comment (alicoding) [inline pages/survey.jsx]: Where do you use this? The language.

**Comment by adamlofting** [comment]
Good point re seperate component. The first attempt was working with their embed code, but this isn't needed with the simple iframe version. I'll tidy up quickly now.

**Comment by alicoding** [comment]
Just few comments but this looks good to me.

**Comment by adamlofting** [comment]
Thanks both, I'm merging this now to get it on staging.

**Pull request merged**